packages/component-environment/__tests__/features/UserIdentity.feature

```diff
@@ -0,0 +1,53 @@
+@author:coleton
+@owner:coleton
+Feature: User identity session management
+
+  Background:
+    Given a user identity context is started
+
+  @motivating
+  Scenario Outline: User identity is initialised based on the environment
+    Given the context is within <SdkEnvironment>
+    When the context is initialised
+    Then is it initialised based on user information with <initialisationSource>
```

Comment:
What is the impact of this @Locrian24? That there is no initialization source for IOS or Andriod?